bool enabled
float32 offset
float32 setpoint
int32 speed
float32 angular_rate
